incubating/newrelic-deployment-marker/package-lock.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
Should this lock file be in the change? I'm guessing it's OK, but I'm not a node.js guy so I figured I'd double check.

Reply:
Yup! package-lock ensures subsequent builds use the same dependency versions